gravitee-apim-console-webui/src/entities/application/application.ts

```diff
@@ -31,7 +31,7 @@ export interface Application {
   status?: string;
   type?: string;
   picture_url?: string;
-  owner?: unknown;
+  owner?: any;
```

Comment:
Can we have a real type ?

Reply:
I didn't want to change too much. I'm afraid of impacts 😅